### ISSUE TITLE
cifar example fails for me

### ISSUE THREAD

**Issue by wingz1** [galeone/dynamic-training-bench#18]
I put this into a script and executed it: (from your website)  except that  
from dytb.inputs import Cifar10  ===>>>     from dytb.inputs.predefined import Cifar10
and
from dytb.models.VGG import  VGG ===>>> from dytb.models.predefined.VGG import VGG
It failed.  Error is below.

import pprint
import tensorflow as tf
from dytb.inputs.predefined import Cifar10
from dytb.train import train
from dytb.models.predefined.VGG import VGG

# Instantiate the model
vgg = VGG()

# Instantiate the CIFAR-10 input source
cifar10 = Cifar10.Cifar10()

# 1: Train VGG on Cifar10 for 50 epochs
# Place the train process on GPU:0
device = '/gpu:0'
with tf.device(device):
    info = train(
        model=vgg,
        dataset=cifar10,
        hyperparameters={
            "epochs": 50,
            "batch_size": 50,
            "regularizations": {
                "l2": 1e-5,
                "augmentation": {
                    "name": "FlipLR",
                    "fn": tf.image.random_flip_left_right
                }
            },
            "gd": {
                "optimizer": tf.train.AdamOptimizer,
                "args": {
                    "learning_rate": 1e-3,
                    "beta1": 0.9,
                    "beta2": 0.99,
                    "epsilon": 1e-8
                }
            }
        })

python cifar10test.py 
Traceback (most recent call last):
  File "cifar10test.py", line 36, in <module>
    "epsilon": 1e-8
  File ".../anaconda3/lib/python3.6/site-packages/dytb/train.py", line 188, in train
    "regularizations"]["augmentation"]["factor"] / args["batch_size"]
KeyError: 'factor'

Any ideas?  Thanks.

**Comment by galeone** [comment]
The post on my website uses a **very old** version of DyTB.

The current version has changed the location of the predefined models and input, as you noticed.
It also includes a new parameter into the "augmentation" section. This parameter is "factor".

"factor" is the number of different samples your augmentation function will generate from a single sample.
In our case, since the augmentation function is just the random flip left / right, the suggested factor to add is 2.

In short, change the "augmentation" section to:
```python
"augmentation": {
    "name": "FlipLR",
    "fn": tf.image.random_flip_left_right,
    "factor": 2
}
```

Let me know if other problems arise

**Comment by wingz1** [comment]
Okay, thanks.  That fixed that problem, but now I have a data problem.  
...
ValueError: Failed to find file:  ...anaconda3/lib/python3.6/site-packages/dytb/inputs/predefined/data/Cifar10/cifar-10-batches-bin/data_batch_1.bin

A listing of the directory shows:  ....anaconda3/lib/python3.6/site-packages/dytb/inputs/predefined/data/Cifar10/
cifar-10-binary.tar.gz
Where/how might I get the cifar-10 data in the proper format for this code to read in?
My main issue with data is that I'm behind a firewall and can't use the standard methods in these codes for grabbing datasets.  (SSLErrors all the time.)

**Comment by galeone** [comment]
When you start the train, does the `>> Downloading cifar-10-binary.tar.gz <percentage>` appears and `<percentage>` reaches 100%?

Because if this happens, your download has been successful and the problem can be in the extraction of the downloaded archive.

If the download fails you have to work around in some way your firewall (use a VPN)

**Comment by wingz1** [comment]
Grabbed the file directly and put it in:
...anaconda3/lib/python3.6/site-packages/dytb/inputs/predefined/data/Cifar10/
170052171 Sep  5 11:37 cifar-10-binary.tar.gz
Ah, if I untar the file, now it seems to be running.  I had a bad file before -- couldn't untar it.
